lib/modules/platform/github/issue.ts

```diff
@@ -0,0 +1,28 @@
+import { z } from 'zod';
+import type { GithubIssue as Issue } from './types';
```

Comment:
I don't understand why and where this is used. 🤔

Reply:
Will be used in upcoming PRs

Reply:
doesn't eslint / typescript errors about unused imports?

Reply:
It's not unused, it's being used in this file and also in the `issueList` field in the `types.ts` itself. But later, `issueList` will go away, so I'll use inference for `GithubIssues`, instead of importing it.